silkrpc/common/log.cpp

```diff
@@ -14,13 +14,52 @@
    limitations under the License.
 */
 
+#include <thread>
+
+#include <absl/time/clock.h>
+
+
 #include "log.hpp"
+#include "tee.hpp"
 
 #include <absl/strings/str_cat.h>
 
 #include <string>
```

Comment:
Include order should be: local module header, standard, third-party, other local headers with full path; within each section lexicographical order. So in this case:
```
#include "log.hpp"

#include <string>
#include <thread>

#include <absl/strings/str_cat.h>
#include <absl/time/clock.h>

#include <silkrpc/common/tee.hpp>
```